### PR TITLE
Update access-cluster-api.md

### PR DESCRIPTION
**Pull request by sb1975**
Cluster-API is different project, hence the correct text here should be Kubernetes API

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by stewart-yu** [comment]
Welcome @sb1975  , thanks for your contribution 👍 
IMO, the `Cluster API` not the project [cluster-api](https://github.com/kubernetes-sigs/cluster-api), just about `K8s cluster API`

**Comment by tengqm** [comment]
Although the cluster-api is a different project, it exposes an API very much like that of the Kubernetes API as I recall. So I'm not sure if this is the correct fix.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-16461--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/access-cluster-api/

**Comment by sb1975** [comment]
@kbhawkey , looks good to me

File: content/en/docs/tasks/administer-cluster/access-cluster-api.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This page shows how to access clusters using the Kubernetes API.
 
 {{% capture steps %}}
 
-## Accessing the cluster API
+## Accessing the Kubernetes API
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/access-cluster-api.md]: @sb1975 . I reread this task page. The page appears to briefly describe several ways a client could communicate with a k8s cluster using the Kubernetes API Server: kubectl, curl or wget, or other programmatic clients. I _think_ the title is somewhat unclear. As far as I can tell, there is no mention of the [cluster-api](https://cluster-api.sigs.k8s.io/) on the task page . 
@tengqm , would it help to change the title (for example: Client Access to the Kubernetes API)? The heading change seems okay.

**Pull request merged**